Updated EmployeeModel

guusjoppe opened the pull request:
Included Skills, Locations and functions in Personal Page and Employees
Works like a charm

guusjoppe committed:
Updated EmployeeModel

Included Skills, Locations and functions in Personal Page and Employees
Works like a charm

guusjoppe committed:
Update Create.cshtml

guusjoppe committed:
Update Edit EmployeeModel

Hicham010 reviewed (approved):
Nice, los je wel effe de merge conflicts op.

guusjoppe committed:
Merge branch 'Guus'

guusjoppe committed:
Merge branch 'master' of https://github.com/jariketting/gride

guusjoppe committed:
Small layout update to Messages

guusjoppe committed:
Merged EmployeeModel en kleine veranderingen in Messages

guusjoppe committed:
Update EmployeeModel

Ervoor gezorgd dat je op de edit pagina de huidige skills, functies en locaties ziet
